### PR TITLE
update deployment docs

### PR DESCRIPTION
**Pull request by jyecusch**

### CODE REVIEW

File: src/pages/guides/getting-started/deployment.mdx [hunk 1]
```diff
@@ -2,42 +2,29 @@ export const description = 'How to deploy Nitric applications'
...
 
-When managing your application with Nitric Deploy, the platform detects the presence of nitric and runs `nitric up` to the cloud you've set up in your Nitric Deploy environment and fully manages your deployment including:
+## Using the official providers
 
-- Integrate with GitHub to connect your repositories
-- Build Nitric Framework applications
-- Clone a sample project to see how it works
-- Configure multiple Stacks for each project (e.g. Production & Staging)
-- Connect & deploy your apps to your own AWS account
+The current providers in nitric for AWS, Google Cloud and Azure all use [Pulumi](https://pulumi.com) under the hood for their deployments, so Pulumi will need to be configured to persist your stack state and to run the deployment to the cloud of your choice.
```
Comment (HomelessDinosaur) [inline src/pages/guides/getting-started/deployment.mdx:11]:
```suggestion
The current providers in Nitric for AWS, Google Cloud and Azure all use [Pulumi](https://pulumi.com) under the hood for their deployments, so Pulumi will need to be configured to persist your stack state and to run the deployment to the cloud of your choice.
```

File: src/pages/guides/getting-started/deployment.mdx [hunk 2]
```diff
@@ -2,42 +2,29 @@ export const description = 'How to deploy Nitric applications'
 
 # Deployment
 
-## Managed Nitric with Deploy
+Nitric can automatically deploy your application to a supported cloud provider. As discussed in the [concepts](/guides/getting-started/concepts) page, Nitric has three standard `providers`, one each for AWS, Microsoft Azure and Google Cloud. If none of those fit your needs you're also able to [build a custom provider](/reference/providers/custom/building-custom-provider) to target the host you prefer with your own tools and configuration.
 
-[Nitric Deploy](https://deploy.nitric.io) is the fastest way to deploy your Nitric application with no configuration.
+The Nitric CLI command to perform deployments is `nitric up`. However, before your first deployment you'll need to create a `stack`, stacks represent deployment targets. If you haven't created a stack before we recommend reading the steps in our [quickstart guide](/guides/getting-started/quickstart#deploying-the-app).
```
Comment (davemooreuws) [inline src/pages/guides/getting-started/deployment.mdx:7]:
```suggestion
The Nitric CLI command to perform deployments is `nitric up`. However, before your first deployment you'll need to create a `stack`, stacks represent deployment targets. If you haven't created a stack before we recommend reading the steps in our [quick-start guide](/guides/getting-started/quickstart#deploying-the-app).
```

File: src/pages/guides/getting-started/deployment.mdx [hunk 3]
```diff
@@ -2,42 +2,29 @@ export const description = 'How to deploy Nitric applications'
...
-- Clone a sample project to see how it works
-- Configure multiple Stacks for each project (e.g. Production & Staging)
-- Connect & deploy your apps to your own AWS account
+The current providers in nitric for AWS, Google Cloud and Azure all use [Pulumi](https://pulumi.com) under the hood for their deployments, so Pulumi will need to be configured to persist your stack state and to run the deployment to the cloud of your choice.
 
-With more on the way...
+### Configuring cloud credentials
 
-## Self-Hosting
+You will need to configure your cloud credentials, either locally or in your CI/CD pipeline, to allow nitric to create resources in a cloud account. Instructions on how to setup credentials for your target provider can be found in our provider documentation: [AWS](/reference/providers/aws), [Google Cloud](/reference/providers/gcp) and [Azure](/reference/providers/azure).
```
Comment (HomelessDinosaur) [inline src/pages/guides/getting-started/deployment.mdx:15]:
```suggestion
You will need to configure your cloud credentials, either locally or in your CI/CD pipeline, to allow Nitric to create resources in a cloud account. Instructions on how to setup credentials for your target provider can be found in our provider documentation: [AWS](/reference/providers/aws), [Google Cloud](/reference/providers/gcp) and [Azure](/reference/providers/azure).
```

**Review by HomelessDinosaur — CHANGES_REQUESTED** [review]

File: src/pages/guides/getting-started/deployment.mdx [hunk 4]
```diff
@@ -2,42 +2,29 @@ export const description = 'How to deploy Nitric applications'
...
+To maintain the state of your deployment between runs you'll need to configure a backend for Pulumi to store its stack state. For this, you can use either [Pulumi's managed service](https://www.pulumi.com/intro/concepts/state/#logging-into-the-pulumi-service-backend), or you could use one of the other state [backends](https://www.pulumi.com/intro/concepts/state/#logging-into-a-self-managed-backend) they provide support for.
 
 ## Configuring Deployment
 
-When deploying the application, you can add extra configuration for specific functions or a global function. This configuration is individualized for each stack file. Below is an example of config added to an AWS stack to augment the memory, timeout, and telemetry sampling percentage.
+When deploying the application, you can add extra configuration for specific handlers or a globally for all handlers. This configuration is unique to each stack file, allowing you to deploy the same application to diffirent environments with different configuration.
```
Comment (davemooreuws) [inline src/pages/guides/getting-started/deployment.mdx:23]:
```suggestion
When deploying the application, you can add extra configuration for specific handlers or a globally for all handlers. This configuration is unique to each stack file, allowing you to deploy the same application to different environments with different configuration.
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Comment by HomelessDinosaur** [comment]
Should also remove the of the environment variable docs that references deploy: https://nitric.io/docs/assets/env#nitric-deploy

File: src/pages/guides/getting-started/deployment.mdx (hunk 2 above)
Comment (jyecusch) [reply src/pages/guides/getting-started/deployment.mdx:7]: Not sure that change is needed the word 'quickstart' is pretty common - e.g. https://docs.github.com/en/get-started/quickstart

**Review by HomelessDinosaur — DISMISSED** [review]

**Review by davemooreuws — DISMISSED** [review]

**Comment by davemooreuws** [comment]
@jyecusch Do you want to add a note to the deploy docs page in case anyone goes to it from google or search bar?

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**